### ISSUE TITLE
Slightly improve select logic

### ISSUE THREAD

**Issue by mattdawkins** [VIAME/VIAME-Web#126]
This is fairly minor because I think it works in most cases, but I hit an issue where it wouldn't switch to the box I wanted.

Reproducible procedure:

Go to: https://viame.kitware.com/#/viewer/5eb371608d01d968cc1642dd

Go to the 2nd frame (with a roundfish and live_sea_scallop)

click the roundfish

click the center of the live_sea_scallop box

it fails to switch to the live_sea_scallop until you select elsewhere

**Comment by BryonLewis** [comment]
Yeah that is a GeoJS limitation in how it decides to select items in the selection API.  I don't know what is technically best solution, I'll discuss it with the team.
[mouseOverOrderClosestBorder](https://opengeoscience.github.io/geojs/apidocs/geo.polygonFeature.html#mouseOverOrderClosestBorder) is the setting in GeoJS.  It prefers to select the item that you are closest to the border with.  You should be able to see it by moving halfway between the roundfish and scallop borders.  On the left side of the halfway point it will select the scallop.  On the right side you are closest to the roundfish border so that would be selected.

Couple of options:

1. At least while inside of a selected object, I process the selection list and make sure it doesn't select the currently selected item.
2. There is a GeoJS option for the highestIndex, but we really don't have these things having different depths to them.
3. Modify geoJS to prefer the item with the closest center, but that makes some awkward interactions if there is one track/detection around the entire frame.  Clicking in the middle will trigger that one.

The best option is probably the first one.

**Comment by subdavis** [comment]
Need to see if this is still relevant.

**Comment by waxlamp** [comment]
I am able to reproduce, but this doesn't strike me as worth the investment to "fix"--it's not clear that clicking on an annotation that is behind the selected annotation should select that one instead. There are other ways to select that one, including clicking on an "exposed" portion of the annotation or using the sidebar to do so.

Can we mark this as `wontfix`?

One way to improve the UX would be to shade the interior of the selected annotation gray, as a subtle clue as to what's "occupying" that space.
